controllers/feeds/getOneFeedChat.js

```diff
@@ -130,6 +124,10 @@ const getOneFeedChatService = async (req, res) => {
 
     const postMakerUserId = data?.results[0]?.actor?.id;
 
+    if (isAnonym && actor) {
+      actor.id = null;
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change makes sense as it ensures that the actor's id is nullified only after all operations involving the actor's id have been completed. However, please ensure that there are no dependencies on the actor's id being nullified earlier in the code.
```diff
    if (isAnonym && actor) {
      actor.id = null;
    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->